app/src/main/res/layout/fragment_feedback_settings.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center"
+    android:orientation="vertical"
+    tools:context=".model.FeedbackSettings">
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingBottom="5dp"
+        android:text="Feedback Settings"
+        android:textAlignment="center"
+        android:textColor="#000"
+        android:textSize="26sp"
```

Comment:
Use values from resources rather than hardcoding them. That makes it easier to have a uniform theme across the entire app.